rpc/gnmi/mixed-schema.md

```diff
@@ -99,6 +99,24 @@ constrained to the specified origin. Particularly:
   the specified `origin`. To delete contents from multiple origins, a client
   MUST specify multiple paths within the `delete` of the `SetRequest`.
 
+### Special considerations for the use of Origin CLI
+* If a `Set` RPC contains `origin` CLI, the CLI origin MUST be given precedence
+  over the OpenConfig origin.  This precedence is independent of the order in
+  which origin CLI appears.  This overrides the gNMI specification requirement
+  of ordered path based precedence within a `SetRequest`.  The goal of this
+  precedence is to create deterministic behavior for resolving overlapping
+  configuration.
```

Comment:
Clarify that for origins _other_ than `cli` this rule continues to be implemented.